Documentation/ExtensionArchitecture/Extbase/Reference/_Annotations/_Lazy.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+namespace T3docs\BlogExample\Domain\Model;
+
+use TYPO3\CMS\Extbase\Annotation\ORM\Lazy;
+use TYPO3\CMS\Extbase\DomainObject\AbstractEntity;
+use TYPO3\CMS\Extbase\Persistence\ObjectStorage;
+
+class Post extends AbstractEntity
+{
+    /**
+     * @var ObjectStorage<Post>
+     */
+    #[Lazy()]
+    public ObjectStorage $relatedPosts;
+    /**
```

Comment:
```suggestion

    /**
```

Empty line betweet the two examples for better readability.